CMakeLists.txt

```diff
@@ -325,9 +325,18 @@ endif()
 
 option(WITH_BLUEFS "libbluefs library" OFF)
 
+option(WITH_SYSTEM_QAT "Use system packages for qat library" OFF)
 option(WITH_QAT "Enable Qat driver" ON)
 if(WITH_QAT)
+  if(NOT WITH_SYSTEM_QAT)
+    include(BuildQAT)
+    build_qat()
```

Comment:
earlier generations of QAT devices (e.g. c62x, dh895xxcc, etc.) are not supported by in-tree qat driver and [qatlib](https://github.com/intel/qatlib). and user need to install out-of-tree qat driver. in this way, there maybe not proper to install qat by ceph.